### PR TITLE
fix: (Core) Add token validate function to MultiInput

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
part of https://github.com/SAP/fundamental-ngx/issues/4483
#### Please provide a brief summary of this pull request.
There is added multi input new token validation method.
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-new-tokens-example/multi-input-new-tokens-example.component.ts [hunk 1]
```diff
@@ -23,8 +23,10 @@ export class MultiInputNewTokensExampleComponent {
...
 
     parseFunc(value: string): object {
-        if (value && value.length) {
-            return { name: value };
-        }
+        return { name: value };
+    }
+
+    validateFunc(value: string): boolean {
+        return value && value.length >= 3
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-new-tokens-example/multi-input-new-tokens-example.component.ts:30]:
```suggestion
        return value?.length >= 3
```

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 2]
```diff
@@ -481,14 +488,21 @@ export class MultiInputComponent implements
         });
     }
 
+    /** @hidden */
     private defaultDisplay(str: string): string {
```
Comment (Betrozov) [inline libs/core/src/lib/multi-input/multi-input.component.ts:492]: please add `_`

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 3]
```diff
@@ -481,14 +488,21 @@ export class MultiInputComponent implements
...
         return str;
     }
 
+    /** @hidden */
     private defaultParse(str: string): string {
```
Comment (Betrozov) [inline libs/core/src/lib/multi-input/multi-input.component.ts:497]: the same

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 4]
```diff
@@ -481,14 +488,21 @@ export class MultiInputComponent implements
...
         return str;
     }
 
+    /** @hidden */
+    private defaultTokenValidate(str: string): boolean {
```
Comment (Betrozov) [inline libs/core/src/lib/multi-input/multi-input.component.ts:502]: the same

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Comment by mikerodonnell89** [comment]
Looks good after addressing @Betrozov 's comments

**Comment by JKMarkowski** [comment]
@Betrozov Thanks for comments, could you recheck it now?

**Review by droshev — APPROVED** [review]

**Pull request merged**